vulture/core.py

```diff
@@ -91,14 +91,22 @@ def _ignore_variable(filename, varname):
         _is_special_name(varname))
 
 
-class Item(str):
-    def __new__(cls, name, typ, filename, lineno, size=1):
-        item = str.__new__(cls, name)
-        item.typ = typ
-        item.filename = filename
-        item.lineno = lineno
-        item.size = size
-        return item
+class Item(object):
+    def __init__(self, name, typ, filename, lineno, size=1):
+        self.name = name
+        self.typ = typ
+        self.filename = filename
+        self.lineno = lineno
+        self.size = size
+
+    def __str__(self):
+        return self.name
+
+    def __eq__(self, other):
+        return self.name == other
```

Comment:
I don't think so. The current code will look like a bug to many people. If we want to compare Item objects to strings, we need to do `item.name == mystring`.